### PR TITLE
rename evmc -> evm

### PR DESCRIPTION
**Pull request by ufoscout**
# Issue ticket

Issue ticket link: <>

### Code conventions

- [x] I have performed a self-review of my code
- [x] Every new function is documented
- [x] Object names are auto explicative

### Security

- [ ] The PR does not break APIs backward compatibility
- [ ] The PR does not break the stable storage backward compatibility

### Testing

- [x] Every function is properly unit tested
- [x] I have added integration tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] IC endpoints are always tested through the `canister_call!` macro

### CODE REVIEW

**Review by itsyaasir — APPROVED** [review]

**Review by GraDKh — APPROVED** [review]

File: src/register-evm-agent/dfx.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
             "wasm": ".artifact/simple_canister.wasm",
             "type": "custom"
         },
-        "evmc": {
-            "candid": "./src/simple_canister/evmc.did",
+        "evm": {
+            "candid": "./src/simple_canister/evm.did",
```
Comment (valeriyr) [inline src/register-evm-agent/dfx.json:10]: can we use a full name here, like evm_canister?

**Review by valeriyr — APPROVED** [review]

File: src/did/README.md [hunk 2]
```diff
@@ -1 +1 @@
-# EVMC DID
+# EVM C DID
```
Comment (flyq) [inline src/did/README.md:1]: maybe `EVM Canister DID`?

**Review by flyq — APPROVED** [review]

File: src/did/README.md (hunk 2 above)
Comment (ufoscout) [reply src/did/README.md]: Done

File: src/register-evm-agent/dfx.json (hunk 1 above)
Comment (ufoscout) [reply src/register-evm-agent/dfx.json:10]: yes, but yesterday we decided to use only evm

**Pull request merged**